nyansat/station/antenny.py

```diff
@@ -1,6 +1,8 @@
 import logging
 
 from config.config import ConfigRepository
+from gps.gps_basic import BasicGPSController
```

Comment:
I would probably want to import these modules only when we want telemetry; the esp32 is very low on memory and it's best for us to import modules only when we really need them